src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,55 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int CAPACITY = 12;
+    private static final int FULL = 11;

```

Comment:
This constant is related to CAPACITY . Changing only one leads to an error (inconsistency).